--- tests/incremental/04-var-rename/09-2_ir_with_changes_1.c
+++ tests/incremental/04-var-rename/09-2_ir_with_changes_1.c
@@ -1,13 +1,14 @@
 void foo() {
-    int fooTwo = 1;
-    fooTwo++;
-    assert(fooTwo == 2);
+    int fooThree = 1;
+    fooThree++;
+    assert(fooThree == 2);
 }
 
 void bar() {
     int barTwo = 10;
-    if (barTwo < 11) barTwo = 20;
-    assert(barTwo == 20);
+    int x = 3;
+    if (x < 11) barTwo = 13;
+    assert(x > 1);
 }
 
 int main() {
