diff --git tests/incremental/03-precision-annotation/02-reluctant-int-annotation.c tests/incremental/03-precision-annotation/02-reluctant-int-annotation.c
index 38187f1c0..698e45b62 100644
--- tests/incremental/03-precision-annotation/02-reluctant-int-annotation.c
+++ tests/incremental/03-precision-annotation/02-reluctant-int-annotation.c
@@ -1,10 +1,11 @@
 #include <goblint.h>

+int f(int in) __attribute__ ((goblint_precision("def_exc", "interval")));
 int f(int in){
   while(in < 17) {
     in++;
   }
-  __goblint_check(in == 17); //UNKNOWN
+  __goblint_check(in == 17);
   return in;
 }

@@ -12,6 +13,6 @@ int main() {
   int a = 0;
   __goblint_check(a); // FAIL!
   a = f(a);
-  __goblint_check(a == 17); //UNKNOWN
+  __goblint_check(a == 17);
   return 0;
 }
