--- tests/incremental/01-global.c	2021-09-29 11:07:05.155250278 +0300
+++ "tests/incremental/01-global copy.c"	2021-09-29 11:07:21.507273052 +0300
@@ -1,7 +1,7 @@
 #include <assert.h>
 #include <pthread.h>
 
-int g = 1;
+int g = 2;
 
 void* t_fun(void *arg) {
     return NULL;
@@ -11,6 +11,6 @@
     pthread_t id;
     pthread_create(&id, NULL, t_fun, NULL); // just go multithreaded
 
-    assert(g == 1); // success before, success after
+    assert(g == 2); // success before, success after
     return 0;
 }
\ No newline at end of file
