--- tests/incremental/00-basic/06-threadid.c
+++ tests/incremental/00-basic/06-threadid.c
@@ -18,6 +18,9 @@ pthread_t create_thread(){
   x = f();
   // Shifting the lines below down in the incremental run should not require a re-evalatuion,
   // but the threadid in the HTML-Output should contain the new location.
+
+
+
   pthread_create(&id, NULL, t_fun, NULL);
   pthread_join (id, NULL);
   x = 0;
