diff --git tests/incremental/00-basic/15-reluctant-test.c tests/incremental/00-basic/15-reluctant-test.c
index eb4c59f6b..5be065852 100644
--- tests/incremental/00-basic/15-reluctant-test.c
+++ tests/incremental/00-basic/15-reluctant-test.c
@@ -14,8 +14,8 @@ int foo(){
 	if(top){
 		r = g;
 	} else {
-		// bar();
-		// r = 5;
+		bar();
+		r = 5;
 	}
 	return r;
 }
@@ -28,7 +28,7 @@ int main(){
 		g = 4;
 		int i = bar();
 
-
+		__goblint_check(i == 4); //UNKNOWN!
 	}
 	return x;
 }
\ No newline at end of file
