diff --git tests/incremental/00-basic/15-reluctant-test.c tests/incremental/00-basic/15-reluctant-test.c
index df4739354..a9117d577 100644
--- tests/incremental/00-basic/15-reluctant-test.c
+++ tests/incremental/00-basic/15-reluctant-test.c
@@ -14,8 +14,8 @@ int foo(){
 	if(top){
 		r = g;
 	} else {
-		// bar();
-		// r = 5;
+		bar();
+		r = 5;
 	}
 	return r;
 }
@@ -29,7 +29,7 @@ int main(){
 		int i = bar();
 
 		// At first unreachable, then UNKNOWN!
-		__goblint_check(i == 4);
+		__goblint_check(i == 4); //UNKNOWN!
 	}
 	return x;
 }
\ No newline at end of file
