--- tests/incremental/05-method-rename/07-common_rename_refactored.c
+++ tests/incremental/05-method-rename/07-common_rename_refactored.c
@@ -1,20 +1,24 @@
 #include<stdio.h>

-void foo() {
+void bar() {
     printf("foo");
 }

+void baz() {
+    printf("baz");
+}
+
 void fun1() {
-    foo();
+    bar();
 }

 void fun2() {
-    foo();
+    bar();
 }

 int main() {
     fun1();
     fun2();
-    foo();
+    baz();
     return 0;
 }
