diff --git tests/incremental/01-force-reanalyze/01-int-reluctant.c tests/incremental/01-force-reanalyze/01-int-reluctant.c
index 38187f1c0..6126fe8cf 100644
--- tests/incremental/01-force-reanalyze/01-int-reluctant.c
+++ tests/incremental/01-force-reanalyze/01-int-reluctant.c
@@ -4,7 +4,7 @@ int f(int in){
   while(in < 17) {
     in++;
   }
-  __goblint_check(in == 17); //UNKNOWN
+  __goblint_check(in == 17);
   return in;
 }

@@ -12,6 +12,6 @@ int main() {
   int a = 0;
   __goblint_check(a); // FAIL!
   a = f(a);
-  __goblint_check(a == 17); //UNKNOWN
+  __goblint_check(a == 17);
   return 0;
 }
diff --git tests/incremental/01-force-reanalyze/01-int-reluctant.json tests/incremental/01-force-reanalyze/01-int-reluctant.json
index d58c2254b..8834d182d 100644
--- tests/incremental/01-force-reanalyze/01-int-reluctant.json
+++ tests/incremental/01-force-reanalyze/01-int-reluctant.json
@@ -2,6 +2,11 @@
     "annotation" : {
         "int" : {
             "enabled" : true
+        },
+        "goblint_precision": {
+            "f": [
+                "interval"
+            ]
         }
     },
     "ana" : {
