--- tests/incremental/11-restart/14-mutex-simple-wrap2.c
+++ tests/incremental/11-restart/14-mutex-simple-wrap2.c
@@ -7,19 +7,19 @@ pthread_mutex_t mutex2 = PTHREAD_MUTEX_INITIALIZER;

 void *t_fun(void *arg) {
   pthread_mutex_lock(&mutex1);
-  myglobal=myglobal+1; // RACE!
+  myglobal=myglobal+1; // NORACE
   pthread_mutex_unlock(&mutex1);
   return NULL;
 }

 void incr() {
-  myglobal=myglobal+1; // RACE!
+  myglobal=myglobal+1; // NORACE
 }

 void wrap() {
-  pthread_mutex_lock(&mutex2);
+  pthread_mutex_lock(&mutex1);
   incr();
-  pthread_mutex_unlock(&mutex2);
+  pthread_mutex_unlock(&mutex1);
 }

 int main(void) {
