diff --git tests/incremental/11-restart/15-mutex-simple-wrap1.c tests/incremental/11-restart/15-mutex-simple-wrap1.c
index 790157639..4e9ef494e 100644
--- tests/incremental/11-restart/15-mutex-simple-wrap1.c
+++ tests/incremental/11-restart/15-mutex-simple-wrap1.c
@@ -17,9 +17,9 @@ void incr() {
 }
 
 void wrap() {
-  pthread_mutex_lock(&mutex2);
+  pthread_mutex_lock(&mutex1);
   incr();
-  pthread_mutex_unlock(&mutex2);
+  pthread_mutex_unlock(&mutex1);
 }
 
 int main(void) {
